custom_model_runner/datarobot_drum/drum/drum.py

```diff
@@ -648,7 +648,10 @@ def _prepare_prediction_server_or_batch_pipeline(self, run_language):
 
     def _needs_class_labels(self):
         if self.target_type == TargetType.BINARY:
-            return self.options.negative_class_label is None or self.options.class_labels is None
+            return (
+                self.options.negative_class_label is None
```

Comment:
do you think we should allow passing in a classLabels.txt for binary target type?

Reply:
This bit of code affect whether the labels are taken from EITHER the class labels file or the unique target values if that is what you are asking, so it can't be removed.  Or are you thinking about changing the logic of `possibly_intuit_order`?